lib/IDE/CodeCompletion.cpp

```diff
@@ -4137,9 +4137,13 @@ class CompletionOverrideLookup : public swift::VisibleDeclConsumer {
     bool missingOverride = Reason == DeclVisibilityKind::MemberOfSuper &&
                            !isKeywordSpecified("override");
 
-    if (missingDeclIntroducer && missingAccess)
-      Builder.addAccessControlKeyword(
-          std::min(VD->getFormalAccess(), AccessibilityOfContext));
+    if (missingDeclIntroducer && missingAccess) {
+      auto Access = std::min(VD->getFormalAccess(), AccessibilityOfContext);
+      // Never emit 'private' for override or conformance decl.
+      if (Access == Accessibility::Private)
+        Access = Accessibility::FilePrivate;
+      Builder.addAccessControlKeyword(Access);
```

Comment:
OK thanks, I will do that.
But, could you merge #4403 first? That will conflict with this change.